docs/backend-api/how-to/how-to-create-a-rule.md

```diff
@@ -0,0 +1,200 @@
+---
+title: How to use rules
+description: Rules used to set up conditions according to which the system logs the events and sends notifications to user.
+---
+
+# How to use rules
+
+Rules used to set up conditions according to which the system logs the events and sends notifications to user.
+
+When a server receives a new portion of data from the device, it checks whether the conditions set are true 
+or false for this data. If they are true, the server generates an event in history, logs it and immediately sends SMS or
+ Email, as specified.
+ 
+### Create

```

Comment:
Maybe not needed to copy&paste the full description of the resource calls in the how-to article? Just add links to API reference.

Reply:
You're right. Thank you. I have changed the article. It seems much better.